installers/install.sh

```diff
@@ -91,10 +117,6 @@ run() {
     #         sudo xattr -d com.apple.quarantine "$native_file"
     #         ;;
     # esac
-
-    echo
-    echo "Successfully installed Tridactyl native messenger!"
-    echo "Run ':native' in Firefox to check."
```

Comment:
I agree that this message is confusing, but I'd rather we improved it rather than removed it totally.

E.g. `Tridactyl native messenger installed; run `:native` in Tridactyl to check that it works`